### PR TITLE
docs: added a sample

### PR DESCRIPTION
**Pull request by ikuleshov**

### CODE REVIEW

**Comment by busunkim96** [comment]
LGTM. Please use `feat` for library features and `fix` for library fixes, as they will trigger version bumps from the release bot.

File: samples/quickstart.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# This application demonstrates the usage of the Analytics Admin API using
```
Comment (busunkim96) [inline samples/quickstart.py:1]: Please add a license header to the top of this file.

Comment (ikuleshov) [reply samples/quickstart.py]: Bu, that would be awesome. Let's setup noxfile/kokoro!

Comment (ikuleshov) [reply samples/quickstart.py]: done

**Comment by ikuleshov** [comment]
Looks like docs-presubmit fails with
ConfigException: Couldn't find build configuration docs-presubmit under /tmp/workspace/workspace/cloud-devrel/client-libraries/python/googleapis/python-analytics-admin/docs/docs-presubmit/src/github/python-analytics-admin/.kokoro/docs.

**Pull request merged**